ansible/playbooks/molecule_docker/molecule/teleop_server_desktop_icons_simulation_haptx_docker/tests/test_teleop_server.py

```diff
@@ -8,7 +8,7 @@
 def test_icons_in_docker(host):
     desktop_path = '/home/' + str(host.user().name) + '/Desktop/'
     script_path = '/home/' + str(host.user().name) + \
-                  '/.shadow_launcher_app/shadow_hand_launcher/'
+                  '/.shadow_launcher_app/shadow_hand_launcher_teleop/'
```

Comment:
is {{container_name}} in this case teleop?

Reply:
would be good then to have the container name as variable and include in the script_path so that its more obvious :) On another note... f-strings? :)

Reply:
These functions aren't directly called so it will have to be hardcoded, but since it uses a default value it will be fine :)

Reply:
but will add fstring